aptos-move/move-examples/mint_nft/sources/minting.move

```diff
@@ -0,0 +1,276 @@
+/// This module is an example of how one can create a NFT collection from a resource account
+/// and allow users to mint from the NFT collection.
+/// Check aptos/move-e2e-tests/src/tests /mint.nft.rs for an e2e example.
+///
+/// - Initialization of this module
+/// Let's say we have an original account at address `0xcafe`. We can use it to call
+/// `create_resource_account_and_publish_package(origin, vector::empty<>(), ...)` - this will create a resource address at
+/// `0b6beee9bc1ad3177403a04efeefb1901c12b7b575ac5124c0205efc0dd2e32a`. The module `mint_nft` will be published under the
+/// resource account's address.
+///
+/// - When using this module, we expect the flow to look like:
+/// (1) call create_resource_account_and_publish_package() to publish this module under resource account's address.
+/// (2) call init_module(): this will retrieve the signer capability of the resource account, and set up the NFT collection to mint.
```

Comment:
Technically this is called as part of publishing the module in step 1

Reply:
ooh yeah you're right, will update the comment. good point